src/game-logic/gameLoop.js

```diff
@@ -0,0 +1,72 @@
+import gameMapCreator from "./gameMap";
+
+const gameMap = gameMapCreator(3);
+
+function updateCells() {
+  const aliveCells = ["x1y0", "x1y1", "x1y2"];
+  aliveCells.forEach(function (currentCell) {
+    gameMap[currentCell].isAlive = true;
+  });
+}
+
+export default function lifeRules() {
+  updateCells();
+  // console.log(gameMap);
+  // to optimize later maybe only worry about alive cells?
+  // Any alive cell that is touching less than two alive neighbours dies.
+  // this will be grabbed in a loop, but for now it will just be set as one cell
+
+  Object.keys(gameMap).forEach(function (currentCell) {
+    const x = gameMap[currentCell].x;
+    const y = gameMap[currentCell].y;
+
+    const noCell = {
```

Comment:
Is this accessible where it's used below?  I think it might need to declared globally no? Or else it's limited to the scope of the `forEach`?